README.md

```diff
@@ -23,270 +25,138 @@ Somesy (**so**ftware **me**tadata **sy**nc) is a CLI tool to avoid messy softwar
-especially **after you significantly modified your project metadata**.
+### Setting up somesy as a pre-commit hook
 
-Note that **changing the ORCID will not be recognized**, because ORCIDs are assumed to be unique per person.
-If you initially have stated an incorrect ORCID for a person and then change it, **somesy will think that this is a new person**.
-Therefore, **in such a case you will need to fix the ORCID in all configured somesy targets** either
-before running somesy (so somesy will not create new person entries), or
-after running somesy (to remove the duplicate entries with the incorrect ORCID).
+We highly recommend to use `somesy` as a [pre-commit hook](https://pre-commit.com/).
+A pre-commit hook runs on every commit to automatically point out issues or fix them on the spot,
+so if you do not use pre-commit in your project yet, you should start today!
+When used this way, `somesy` can fix most typical issues with your project
+metadata even before your changes can leave your computer.
 
-<!-- --8<-- [end:quickstart] -->
+To add `somesy` as a pre-commit hook, add it to your `.pre-commit-config.yaml`
+file in the root folder of your repository:
 
-## Development
+```yaml
+repos:
+  # ... (your other hooks) ...
+  - repo: https://github.com/Materials-Data-Science-and-Informatics/somesy
+    rev: "0.1.0"
+    hooks:
+      - id: somesy
+```
 
-This project uses [Poetry](https://python-poetry.org/) for dependency management,
-so you will need to have it
-[installed](https://python-poetry.org/docs/master/#installing-with-the-official-installer)
-for a development setup for working on this package.
+Note that `pre-commit` gives `somesy` the [staged](https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F) version of files,
+so when using `somesy` with pre-commit, keep in mind that
 
-Then you can run the following lines to setup the project:
+* if `somesy` changed some files, you need to `git add` them again (and rerun pre-commit)
+* if you explicitly run `pre-commit`, make sure to `git add` all changed files (just like before a commit)
 
-```bash
-$ git clone git@github.com:Materials-Data-Science-and-Informatics/somesy.git
-$ cd somesy
-$ poetry install
-```
+## Supported File Formats
 
-Common tasks are accessible via [poethepoet](https://github.com/nat-n/poethepoet),
-which can be installed by running `poetry self add 'poethepoet[poetry_plugin]'`.
+Here is an overview of all the currently supported files and formats.
 
-- Use `poetry poe init-dev` after cloning to enable automatic linting before each commit.
+| Input Formats  | Status | | Target Formats                | Status |
+| -------------- | ------ |-| ----------------------------- | ------ |
+| (.)somesy.toml | ✓      | | pyproject.toml _(poetry)_     | ✓      |
+| pyproject.toml | ✓(1.)  | | pyproject.toml _(setuptools)_ | ✓(2.)  |
+| package.json   | TBD    | | package.json                  | TBD    |
```

Comment:
Should we have a package.json input format? Beforehand we talked we can but I am not sure if it is necessary